language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSClientCapabilitiesImpl.java

```diff
@@ -141,6 +141,11 @@ private InitializationOptions parseInitializationOptions(Map<String, Object> ini
                 Boolean.parseBoolean(String.valueOf(semanticTokensSupport));
         initializationOptions.setEnableSemanticTokens(enableSemanticTokens);
 
+        Object renameSupport = initOptions.get(InitializationOptions.KEY_RENAME_SUPPORT);
+        boolean enableRenameSupport = renameSupport == null ||
```

Comment:
Are we providing rename support by default when the `renameSupport` is null (client has not specified init option) @IMS94

Reply:
No. We should not. We should consider that capability is not available if this is null (to be compatible with old ballerina versions)